djangular/forms/angular_model.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, data=None, *args, **kwargs):
             self.ng_directives['ng-model'] = '%(model)s'
         self.prefix = kwargs.get('prefix')
         if self.prefix and data:
-            data = dict((self.add_prefix(name), value) for name, value in data.get(self.prefix).items())
+            data = {name: value for name, value in data.items() if name.startswith(self.prefix + '.')}
```

Comment:
Dictionary comprehensions were introduced in Python 2.7. If you wish to support previous versions of Python, then `dict()` is definitely a better choice. (However, I'm taking it from setup.py that you're only supporting 2.7+, yes?)